src/@codegen/meteora/instructions/addLiquidityByStrategy.ts

```diff
@@ -38,7 +38,7 @@ export interface AddLiquidityByStrategyAccounts {
   program: Address
 }
 
-export const layout = borsh.struct([
+export const layout = borsh.struct<AddLiquidityByStrategyArgs>([
```

Comment:
```suggestion
export const layout = borsh.struct([
```

Manually fix this for now there is a bug in codegen lib